### ISSUE TITLE
Copy contact info to clipboard on sharing

### ISSUE THREAD

**Issue by vkopichenko** [johanhil/copy-to-clipboard#1]
It's a feature request.
Currently when trying to share contact to this app the clipboard is just cleared. (Nexus One, Froyo)
It would be very nice if common contact info (names, numbers, addresses, etc) copied to clipboard in such case.
This is useful for sending chosen contact info as plain text in sms or elsewhere.

**Comment by johanhil** [comment]
Indeed. This would be very useful. Unfortunately to access the contact's information, I'd need Share to Clipboard to have permission to do this. I'm still on the fence if this is needed or if a simple "nope, can't copy contacts" popup warning is a better solution. Hm.

**Comment by kashifo** [comment]
i would like to do that
